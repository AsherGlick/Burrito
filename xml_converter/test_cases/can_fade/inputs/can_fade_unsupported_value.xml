<OverlayData>
<MarkerCategory  Name="unsupportedvalue">
</MarkerCategory>

<POIs>
<POI  CanFade="yes" Type="unsupportedvalue"/>
</POIs>
</OverlayData>
