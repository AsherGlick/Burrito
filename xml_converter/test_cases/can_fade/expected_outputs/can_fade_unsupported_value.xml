<OverlayData>
<MarkerCategory  Name="unsupportedvalue">
</MarkerCategory>

<POIs>
<POI  CanFade="false" Type="unsupportedvalue"/>
</POIs>
</OverlayData>
